src/domain/QuestionList/QuestionListCard/QuestionListCardSmall.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import styles from './QuestionListCard.module.scss';
+import { QuestionListCardSmallProps } from './QuestionListCardSmallProps';
+
+export default function QuestionListCardSmall(
+  props: QuestionListCardSmallProps
+): JSX.Element {
+  const { list, categories, onCardClickedCallback } = props;
+
+  return (
+    <div
+      role="presentation"
+      onClick={onCardClickedCallback}
+      onKeyDown={() => {
```

Comment:
Is this method declaration needed since it does nothing?

Reply:
Eslint rule workaround seems better than disabling it altogether